scripts/installdeps.ps1

```diff
@@ -0,0 +1,15 @@
+<#
+Copyright (c) AppDynamics, Inc., and its affiliates 2015
+All rights reserved
+
+Installation script to setup dependencies using the Chocoloatey package manager.
+#>
+
+# Install choco
+iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))
```

Comment:
Are the log messages and output from this command sufficient to debug if something goes wrong? Say, for example, chocolatey.org is down?